docs/release-notes/v5.stories.mdx

```diff
@@ -0,0 +1,75 @@
+import { Meta } from '@storybook/addon-docs';
+
+<Meta
+	title="Release notes/Source v5"
+	parameters={{ previewTabs: { canvas: { hidden: true } } }}
+/>
+
+# What's new in v5.0.0
+
+_tl;dr_
+
+-	Accessibility improvements (some components look different)
+-   Component APIs have not changed
+-   Functionality and dependencies have not changed
+
+This brings Source closer to WCAG 2.1 Level AA compliance.
+
+## Migrating to v5
+
+From a code perspective, v5 of Source is a drop-in replacement for v4. There are no
+breaking API changes and functionality has not changed.
+
+There are a number of updates that change the way Source components appear. There are
+also some colour changes. 
+
+Consumers are advised to ensure that the updated Source components and colours still work 
+well with their products.
+
+If a changed component or colour appears in automated visual regression tests (e.g. in
+Chromatic), please review the changes before upgrading Source.
+
+## Changes
+
+### Identifying Links and Buttons
+
+Links and Buttons are distinctively identified as interactive elements by applying
+ an underline to the text.
+
+- Deprecate `subdued` styling in `<Link>` component
+- Give all links (including now-deprecated `subdued`) an underline.
+- Increase the thickness of the `<Link>` component's underline on hover
+- Give text-only `<Button>` components (`priority="subdued"`) an underline which increases in thickness on hover
+- Add 3px of space to focused `<Button>` components a slightly padded focus halo
```

Comment:
Does this make sense? I'm finding it hard to explain #1432 in a snappy way! 😅

```suggestion
- Add a small amount of spacing between the `<Button>` and the focus outline
```

Reply:
very much so